src/Resources/contao/templates/survey/survey_answers_matrix.html5

```diff
@@ -10,11 +10,25 @@
 </thead>
 <tbody>
 <?php $colcounter = 1; ?>
-<?php foreach ($this->choices as $choiceid => $choice): ?>
+<?php foreach ($this->choices as $choiceid => $choice):
+dump(array_key_exists($counter, $this->statistics['cumulated']));
+?>
 	<tr>
 		<td class="counter"><?php echo $colcounter; ?>.</td>
 		<td class="answer"><?php echo $choice; ?></td>
-		<td class="selections"><?php echo (($this->statistics['cumulated'][$counter][$colcounter]) ? $this->statistics['cumulated'][$counter][$colcounter] : 0); ?></td>
+		<td class="selections">
```

Comment:
Couldn't this be replaced with `echo ($this->statistics['cumulated'][$counter][$colcounter] ?? '?')` ?

Reply:
Yes, possibly. But I would have to take a closer look. The errors and warnings don't come because the value of the "addressed cell" in the array is null, but because the keys are not defined. This was probably different before PHP 8. 

Therefore it seems to me that '??' in 

$this->statistics['cumulated'][$counter][$colcounter] ?? '?'

is not the optimal solution here, since the warning already occurs if 'cumulated' (means here $this->statistics['cumulated'] = null) or $counter or $colcounter is null or empty or undefined. I am not sure though...

Reply:
OK - yes - we can use your suggestion.

Reply:
null coalescing operator works also on multi dimensional array: https://onlinephp.io?s=s7EvyCjg5VIAApXEoqLESgVbhWgIHwTUE9UVbO0UotWToHQymFZPUY-NhaiKtYbQqckZ-QoaEDOigdpiQXpiQRpiFeztFdTzSnNy1DX1lGLylKwVsGmpwKsFpyWVuHXY2wEA&v=8.2.5%2C7.4.33

Reply:
Yes - that is true - but please see my comments again: https://github.com/pdir/contao-survey/pull/36#discussion_r1169795737